web/libs/editor/src/components/ImageTransformer/ImageTransformer.jsx

```diff
@@ -214,10 +225,14 @@ export default class TransformerComponent extends Component {
           }}
           dragBoundFunc={this.dragBoundFunc}
           onDragEnd={() => {
+            // Call applyTransform for VectorRegion instances
```

Comment:
Move this comment to the method itself. Why? Here it reads naturally and feels a good fit, you don't need to describe it using one specific tag. But the method itself could use a description saying that it's applied just to one specific tag and also with explanation why only for this one tag.